.github/workflows/.build.yaml

```diff
@@ -33,5 +34,6 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           docker login ghcr.io -u ${GITHUB_REPOSITORY} -p ${GITHUB_TOKEN}
-          docker build --tag ${{ env.image }} -f docker/Dockerfile apps/${{ github.workflow }}
+          docker build --tag ${{ env.image }} --tag ${{ env.branch-image }} -f docker/Dockerfile apps/${{ github.workflow }}
```

Comment:
Hvorfor trenger vi å tagge imaget for å deploye main?

Reply:
Den la oss deploye sist bygd main - ikke bygg på nytt. Hvis vi skal ha nytt bygg er det vel bedre å pushe det som vi vil ha med?

Reply:
Jeg som så meg blind på diffen, høres bra ut!